frontend/src/modules/EditZing/Components/GroupGrid.tsx

```diff
@@ -33,7 +35,16 @@ export const GroupGrid = ({
         style={{ opacity: isOver ? '0.6' : '1' }}
       >
         <StyledGroupTextWrapper>
-          <StyledGroupText>{'Group ' + String(groupNumber)}</StyledGroupText>
+          <Tooltip
+            disableFocusListener
+            disableTouchListener
+            title={
+              /* Could use getMonth and getDate to format mm/dd */
+              'Created on ' + createTime.toLocaleDateString()
```

Comment:
Do you know what exact format you need? I believe that if you want mm/dd exactly you should change it to what you have in your comment, but also I'm not sure if this edge case would occur (most of this would happen in the US I would assume).